kDriveFileProvider/Enumerators/WorkingSetEnumerator.swift

```diff
@@ -21,27 +21,33 @@ import InfomaniakDI
 import kDriveCore
 
 final class WorkingSetEnumerator: NSObject, NSFileProviderEnumerator {
-    @LazyInjectService var additionalState: FileProviderExtensionAdditionalStatable
-
     let driveFileManager: DriveFileManager
+    let workingSetService: FileProviderWorkingSetServiceable
     let domain: NSFileProviderDomain?
 
     init(driveFileManager: DriveFileManager, domain: NSFileProviderDomain?) {
         self.driveFileManager = driveFileManager
+        let userId = driveFileManager.drive.userId
+        let driveId = driveFileManager.drive.id
+        let metadata = ["driveId": driveId, "userId": userId]
+        @InjectService(factoryParameters: metadata) var workingSetService: FileProviderWorkingSetServiceable
+        self.workingSetService = workingSetService
         self.domain = domain
     }
 
     func invalidate() {}
 
     func enumerateItems(for observer: NSFileProviderEnumerationObserver, startingAt page: NSFileProviderPage) {
+        Log.fileProvider("•• WorkingSet enumerate")
```

Comment:
I cannot get the iPhone XR (ios7.3) to call this method and display a single file.
Fighting with Spotlight indexing, will check project setup at this point.